### DISCUSSION TITLE
What is Continuous Integration (CI)?

### DISCUSSION THREAD

**Discussion by PPartisan** [PPartisan/Roky#25] [Q&A]
The project uses Continuous Integration (CI), which means several actions run whenever someone opens a pull request. What is CI, and why is it important? 

This project uses GitHub Actions: Are there any resources I can use to learn more about working with GitHub Actions and adapting the CI of this project?

**Comment by second-ed** [comment]
Some good stuff explaining git actions here:
https://github.blog/enterprise-software/ci-cd/build-ci-cd-pipeline-github-actions-four-steps/
